auslib/db.py

```diff
@@ -1014,7 +1030,7 @@ def update(self, where, what, changed_by, old_data_version, transaction=None, dr
                     base_columns[base_col] = base_what[base_col]
                 elif sc_columns.get(col):
                     base_columns[base_col] = sc_columns[col]
-
+            base_columns["change_type"] = sc_columns["change_type"]
```

Comment:
Is this necessary? I'm surprised that it doesn't happen as part of the if/elif block above.

I see you're doing it in update/delete/enactChange as well. Can you add a comment explaining why? It looks a bit odd to be copying one specific column after copying a bunch of others in a loop or comprehension.